python/cudf_polars/tests/test_scan.py

```diff
@@ -86,3 +87,12 @@ def test_scan(df, columns, mask):
     if columns is not None:
         q = df.select(*columns)
     assert_gpu_result_equal(q)
+
+
+def test_scan_unsupported_raises(tmp_path):
+    df = pl.DataFrame({"a": [1, 2, 3]})
+
+    df.write_ndjson(tmp_path / "df.json")
+    q = pl.scan_ndjson(tmp_path / "df.json")
+    with pytest.raises(NotImplementedError):
+        _ = translate_ir(q._ldf.visit())
```

Comment:
It's a little unclear what we're invoking on this line and expecting to fail. Is it the translation of the plan to IR? If this is a pattern we might use in the testing codebase, can we have a utility like `assert_gpu_result_equal` that we use that says what this is trying to do, either in the name or the docs?

Reply:
Good idea

Reply:
Done.